apps/worker/worker_core/tasks/transcribe.py

```
@@ -0,0 +1,10 @@
+from worker_core.main import app
+from worker_core.config import OPENAI_API_KEY
+from fastfetchbot_file_export.transcribe import get_audio_text
+
+
+@app.task(name="file_export.transcribe")
+def transcribe_task(audio_file: str, openai_api_key: str = None) -> dict:
```

Ruff (0.15.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)